CHANGES.md

```diff
@@ -1,38 +1,97 @@
+# 2.17.0 - 26 Sep 2024
+
+## Features
+
+* optionally install Scenario with `ops[testing]` and expose the names in ops.testing in (#1381)
```

Comment:
This is probably missing from HACKING.md: we put a capital letter for each of the bullet points (but annoyingly conventional commits wants a lower-case letter, so it's a manual job).

(Same comment for all the bullets).

Reply:
And here I went and corrected some of my accidentally capitalised PRs to lowercase to match the others ...

Reply:
> And here I went and corrected some of my accidentally capitalised PRs to lowercase to match the others ...

Maybe we should ignore the conventional commits rule going forward and have "feat: Adds a new Foo to the Bar class"? I like that more, personally, and it makes this bit a little less tedious.

I say this every month, but I also need to remember to think about "reviewing" the PR title when I do reviews so that I suggest improvements then rather than when we come to the release.